spec/statement.md

```diff
@@ -0,0 +1,131 @@
+* [Syntax](#syntax)
+  * [ScssStatements](#scssstatements)
+  * [IndentedStatements](#indentedstatements)
+  * [Statements](#statements)
+  * [Block](#block)
+  * [Whitespace](#whitespace)
+  * [Indentation](#indentation)
+
+## Definitions
+
+### Current indentation level
+
+The `current indentation level` is the count of the [`documentation indentation
+character`] between a new line and any non-whitespace characters of the last
+consumed line.
+
+A document parsed in the indented syntax may not begin with whitespace, and the
+initial `current indentation level` for any document is `0`.
+
+Changes in the indentation level are used by the indented syntax to start and
+end blocks of statements.
+
+[`documentation indentation character`]: #document-indentation-character
+
+### Document indentation character
+
+The `document indentation character` is the character used for calculating the
+[`current indentation level`], and is either a space or tab character.
+
+[`current indentation level`]: #current-indentation-level
+
+It is defined as the first tab or space character used as an [IndentMore]
+production in a document.
+
+In the indented syntax, no character other than the `document indentation
+character` may be used for indentation.
+
+## Syntax
+
+### ScssStatements
+
+<x><pre>
+**ScssStatements**      ::= (Statement ';'?¹)*Statement?
+</pre></x>
+
+1: This production is mandatory unless the previous `Statement` is a
+`LoudComment` or `SilentComment`, or after the final production in a
+`ScssStatements` production, which is either at the end of a `Block` or the end
+of the document.
+
+If a `WhitespaceComment` would be ambiguous with a `Statement` in the
+`ScssStatements` rule, parse it preferentially as a `Statement`.
+
+### IndentedStatements
+
+<x><pre>
+**IndentedStatements**  ::= (Statement IndentedSame?¹)* Statement?
+</pre></x>
+
+1: This production is mandatory unless the previous `Statement` is a
+`LoudComment` or `SilentComment`, or after the final production in a
+`IndentedStatements` production, which is either at the end of a `Block` or the
+end of the document.
+
+If a `WhitespaceComment` would be ambiguous with a `Statement` in the
+`IndentedStatements` rule, parse it preferentially as a `Statement`.
+
+### Statements
+
+<x><pre>
+**Statements**          ::= [ScssStatements] | [IndentedStatements]¹
+</pre></x>
+
+[ScssStatements]: #scssstatements
+[IndentedStatements]: #indentedstatements
+
+Only the production for the current syntax is valid.
+
+### Block
+
+<x><pre>
+**ScssBlock**      ::= '{' [ScssStatements] '}'
+**IndentedBlock**  ::= [IndentMore] [IndentedStatements] [IndentLess]
+**Block**          ::= ScssBlock | IndentedBlock¹
+</pre></x>
+
+[IndentMore]: #indentation
+[IndentLess]: #indentation
+
+1: Only the production for the current syntax is valid.
+
+### Whitespace
+
+> Whitespace separates productions inside a statement when disambiguation is
+> necessary. The Whitespace productions do not separate Statements.
```

Comment:
Several issues here:

* This seems like it's supposed to be normative, but it's in a block quote which indicates descriptive text.
* What does "inside a statement" mean? Block productions and thus the `Statements` production are also inside statements.
* When is disambiguation necessary? Does this mean that whitespace isn't allowed any time it wouldn't strictly disambiguate something?
* Statements certainly _are_ separated by whitespace in SCSS. `@foo; @bar` is a valid stylesheet with whitespace between the statements.

Reply:
This was indeed meant to be a non-normative comment to remind myself to consider the difference between subtraction and unary negation, for example, and the difference between Whitespace and Indentation. It's not needed anymore.